### ISSUE TITLE
bug(android/engine): sil_euro_latin keyboard crashing on KAB 11.0.4

### ISSUE THREAD

**Issue by darcywong00** [keymanapp/keyman#9983]
From  community forum
https://community.software.sil.org/t/keyman-keyboard-not-working-in-my-dictionary-app/6070/4

A user reported the current sil_euro_latin keyboard (2.0.4) is crashing when attempting to install with [KAB 11.0.4](https://software.sil.org/downloads/r/keyboardappbuilder/Keyboard-App-Builder-11.0.4-Setup.exe) which uses Keyman Engine for Android 16.0.142.

As a workaround, the user can use basic_kbdus.

Currently, KAB is still extracting keyboard packages and using just the .js files.

**Comment by darcywong00** [comment]
When I repro'd the issue on my device, I saw the following console error:
```
Cannot find the EuroLatin (SIL) keyboard for English at /data/user/...app_data/cloud/sil_euro_latin-null.js
```

then though the apk file contains assets/cloud/sil_euro_latin-2.0.4.js. Waiting on hearing from the KAB team why the version isn't detected.

**Comment by darcywong00** [comment]
As a workaround, I was able to "rename" the internal files of sil_euro_latin.kmp into "sil_euro_latin2.kmp", and that worked with KAB.

There's likely some lingering bug in determining the keyboard version of `assets/cloud/sil_euro_latin-x.y.z.js` (sil_euro_latin expected as a default fallback in `assets/packages/sil_euro_latin/`.

Keyman Engine for Android deprecated cloud/ keyboards several versions ago, so maybe this is "Won't fix"?

**Comment by darcywong00** [comment]
From A17S27 sprint plan, won't fix in Keyman. I'll mark an issue for KAB to track
